packages/system-tests/src/helpers/screenshot.ts

```diff
@@ -40,7 +40,11 @@ export class Screenshot {
         : `${this.counter++}.png`
     );
 
-    const image = await this.spectron.app.browserWindow.capturePage();
+    const image = await this.spectron.app.browserWindow
+      .capturePage()
```

Comment:
`capturePage` is using `toPng` which is not available in electron 2.0+ which we are on after the vscode update. A fix has been merged to spectron (electron/spectron#297) but hasn't been released yet so I'm adding a catch to stop tests from failing.

Reply:
Sounds reasonable.